CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Git LFS Changelog
 
+## v0.6.0 (10 September, 2015)
+
+First release with full support for the Batch API, while still supporting
+servers with the Legacy API. Also introduces new fetch/checkout/push commands.
+
+### Fetch
+
+* Rename old `git lfs fetch` command to `git lfs pull`. #527 (@sinbad)
+* Add `git lfs checkout` #527 #543 #551 #566 (@sinbad)
+* Add `git lfs fetch` for _just_ downloading objects. #527 (@sinbad)
+  * Support fetching multiple refs #542 (@sinbad)
+  * Add `--all` option for download all objects from the server. #633 (@sinbad)
+  * Add `--include` and `--exclude` flag for `git lfs fetch` #573 (@sinbad)
+* Fix error handling while `git update-index` is running. #570 (@rubyist)
```

Comment:
Also default remote derivation for fetch/pull has slightly changed, it's no longer always `origin`. It follows `git pull` and respects tracking branches.